peewee_migrate/cli.py

```diff
@@ -85,15 +85,29 @@ def create(name, database=None, auto=False, auto_source=False, directory=None, v
 
 
 @cli.command()
-@click.argument('name')
+@click.argument('name', required=False)
+@click.option('--count', required=False, default=1, type=int)
```

Comment:
Add help text which describe sitation when it will be ignored (`name` arg specified)